src/test/java/hudson/plugins/git/UserRemoteConfigRefSpecTest.java

```diff
@@ -0,0 +1,21 @@
+package hudson.plugins.git;
+
+import hudson.util.FormValidation;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class UserRemoteConfigRefSpecTest {
+
+    @Issue("JENKINS-57660")
+    @Test
+    public void testdoCheckRefspec(){
+        String url = "https://github.com/daytonpa/pipeline_test_example.git";
```

Comment:
A more robust choice of a repository might be better, perhaps this repository itself is a good candidate.

Reply:
@res0nance since I believe that the repository need not exist for this test, it might be better to name it `git://git.example.com/repository-that-does-not-exist`.  That would make it very clear it is not a real repository.

Reply:
Okay, I will change the URL as I rewrite the test.